payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherViewModel.kt

```diff
@@ -49,7 +51,8 @@ internal class GooglePayLauncherViewModel(
     private val paymentController: PaymentController,
     private val googlePayJsonFactory: GooglePayJsonFactory,
     private val googlePayRepository: GooglePayRepository,
-    private val savedStateHandle: SavedStateHandle
+    private val savedStateHandle: SavedStateHandle,
+    private val errorReporter: ErrorReporter
```

Comment:
I find it nice to add a trailing comma in these scenarios. It makes the diff easier to read the next time someone adds a param to it.

Reply:
nice, thanks for calling this out!